tests/Analytics/AnalyticsTest.php

```diff
@@ -24,19 +25,8 @@ class AnalyticsTest extends TestCase
     /** @var \Utopia\Analytics\Adapter\Orbit */
     public $orbit;
 
-    public function __construct()
-    {
-        parent::__construct();
-        $this->ga = new GoogleAnalytics(App::getEnv('GA_TID'), App::getEnv('GA_CID'));
-        $this->ac = new ActiveCampaign(
-            App::getEnv('AC_KEY'),
-            App::getEnv('AC_ACTID'),
-            App::getEnv('AC_APIKEY'),
-            App::getEnv('AC_ORGID')
-        );
-        $this->pa = new Plausible(App::getEnv('PA_DOMAIN'), App::getEnv('PA_APIKEY'), 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36', '192.168.0.1');
-        $this->orbit = new Orbit(App::getEnv('OR_WORKSPACEID'), App::getEnv('OR_APIKEY'), 'Utopia Testing Suite');
```

Comment:
Why don't we need this anymore?

Reply:
It's already in the `setUp` method of the test. So it was redundant.